lib/tasks/s3_file_conversion.rake

```diff
@@ -0,0 +1,39 @@
+require 'fileutils'
+include CarrierWave::MiniMagick
+
+namespace :s3 do
+  # Usage:
+  #     rake s3:file_conversion
+  desc "convert manually uploaded files to carrierwave objects"
+  task file_conversion: :environment do |task, args|
+    records = []
+
+    [Collection, Transcript].each do |klass|
+      klass.where.not(image_url: nil).find_each do |resource|
+        image_url = resource.read_attribute(:image_url)
+
+        if image_url.present? && resource.image.blank?
+            records << resource
+
+            dir_name = FileUtils.mkpath('tmp/s3_uploads/')
+            file_name = File.basename(image_url)
+            file_path = File.join(dir_name, file_name)
+
+            # Download the image from S3 and store in temp folder.
```

Comment:
The tmp file will be cleared out with the next deployment, and there are only 195 files.
I thought it might be a nice way to check all the files that had been converted because S3 won't really change.